src/List/ItemList.js

```diff
@@ -1,13 +1,27 @@
-import React, { useState, useEffect, Fragment } from "react";
+import React, { useEffect, Fragment, useState } from "react";
 import { NavLink } from "react-router-dom";
 import "firebase/firestore";
 import * as firebase from "../lib/firebase";
-import classes from "./List.module.css";
+import Modal from "../Modal/Modal";
```

Comment:
I really like that you all used Modal - I think it makes for a cleaner UI experience.